src/main/java/at/hannibal2/skyhanni/utils/repopatterns/RepoPatternManager.kt

```diff
@@ -169,7 +169,8 @@ object RepoPatternManager {
      */
     private fun reloadPatterns() {
         val remotePatterns = remotePattern
-        for (it in usedKeys.values) {
+        val instances = usedKeys.values
+        for (it in instances) {
```

Comment:
This does not change anything. The right hand side of `in` is only evaluated once.